subworkflows/local/standardisation_profiles.nf

```diff
@@ -117,7 +117,19 @@ workflow STANDARDISATION_PROFILES {
                                     [[id:it[0]], it[1]]
                                 }
 
-    KAIJU_KAIJU2TABLE_COMBINED ( ch_profiles_for_kaiju, ch_input_databases.kaiju.map{it[1]}, params.kaiju_taxon_rank)
+    ch_input_for_kaiju2tablecombine = ch_profiles_for_kaiju
+                                        .dump(tag: "B41")
+                                        .map { meta, profile -> [meta['id'], meta, profile] }
+                                        .dump(tag: "B42")
+                                        .combine(ch_input_databases.kaiju.map{meta, db -> [meta.db_name, meta, db]}, by: 0)
```

Comment:
I prefer the `.key` way to the `['key']` of accessing map values myself, but I feel it should be done consistently.

Reply:
Agreed! Have gone with the shorter version :+1: